libr/core/cmd_eval.c

```diff
@@ -235,7 +235,7 @@ static int cmd_eval(void *data, const char *input) {
 					eprintf ("Something went wrong\n");
 				}
 			} else if (input[2]=='?') {
-				eprintf ("Usage: eco [themename]  ;load theme from /usr/share/radare2/0.10.2-git/cons/\n");
+				eprintf ("Usage: eco [themename]  ;load theme from /usr/share/radare2/0.10.4-git/cons/\n");
```

Comment:
Wtf srsly? There's last/ whichc is a symlinl... But cmon... This path is not hardcoded, you have vars to grab the prefix, datadir, destdir...